internal/runners/ppm/convert_test.go

```diff
@@ -0,0 +1,183 @@
+package ppm
```

Comment:
Most of these tests read like integration tests to me, not unit tests. We invested heavily in writing an integration testing framework so please use that instead ;) Unit tests should be testing single functions. If you're relying on channels for tests it's a good signal you should be integration testing instead.

Reply:
With the removal of the interrupt-signal-handling, it should have improved, and be clearer that these are actually unit tests.